### ISSUE TITLE
Can't capture anything except visible part of the page when inside Gmail

### ISSUE THREAD

**Issue by carlbennett** [rojer/a-s-minus#29]
Only the 'capture visible part of the page' feature works when I am on a tab that has Gmail loaded. Doesn't matter what page of Gmail it is, even if I open an email and go to the printer-friendly version.

I want to use Awesome Screenshot Minus to capture entire Gmail threads sometimes and the printer-friendly version of email threads doesn't even let me do that with this extension; I have to either have the window large enough for the visible part of the page feature to capture it or I have to stitch it together myself in photo editing software.

**Comment by jkayani** [comment]
Hey:

I've looked into this issue, and it looks like it's related to #14. If you follow the advice I gave in there, 

> Until this is addressed, if you really have to take selective screenshots in Gmail, you may want to just comment out line 447 in content_script.js and load the extension yourself. This worked fine for me.

The "Capture Entire Page" button works exactly like the "Capture Visible Area" button. Definitely progress from just hanging on the "Capturing..." popup, but not desired behavior. 

I'll look into this further; just to confirm, if you try the "Capture Selected Area" button in Gmail, just like what the person from #14 tried, can you reproduce his error (i.e, nothing happens)?

**Comment by carlbennett** [comment]
Yes, that is correct. Trying to captured a selected area also fails.

![image](https://cloud.githubusercontent.com/assets/592196/16903301/4cb2a85c-4c40-11e6-9d4f-d339458d2154.png)

The above is the error message(s) that appear after selected 'Capture Selected Area'.

**Comment by jkayani** [comment]
Progress update: If you try the suggestion I made earlier (commenting out a line in `content_script.js`), and you use the printer-friendly version of a Gmail thread, the screenshot includes more of the entire page. Unfortunately, there are lot of cases where it doesn't capture the whole thing - I'm not entirely sure why. 

The problem with taking the screenshot from the regular Gmail page appears to be with how emails are formatted in the HTML, so my primary goal will be for email thread screenshots to work from the printer-friendly interface. There might be a need for a Gmail special case to get email thread screenshots working from the regular interface.
